### ISSUE TITLE
Publishing date

### ISSUE THREAD

**Issue by VarnithaKurliReddy** [jataware/covid-19-data#3]
I was wondering how I should make sense of the publishing date?

**Comment by brandomr** [comment]
Hi @VarnithaKurliReddy that is based on a machine extraction from the articles we collected.

**Comment by VarnithaKurliReddy** [comment]
Thanks @brandomr ! Could you please tell me how I can interpret say "43908.0" in row 2.

**Comment by VarnithaKurliReddy** [comment]
To elaborate more, what we are looking for is the actual date when the NPI mandates were implemented across counties? Does publishing date refer to that date?

**Comment by brandomr** [comment]
No, that was more likely the date the article was published. This entire dataset was automatically generated.

For higher fidelity data I'd check out [Keystone NPI data](https://www.keystonestrategy.com/coronavirus-covid19-intervention-dataset-model/]) or [National League of Cities NPI Data](https://covid19.nlc.org/resources/covid-19-local-action-tracker/)
